src/validation/rules/UniqueDirectivesPerLocation.js

```diff
@@ -48,7 +70,11 @@ export function UniqueDirectivesPerLocation(
               ]),
             );
           } else {
-            knownDirectives[directiveName] = directive;
+            const repeatable = repeatableMap[directiveName];
```

Comment:
Can you just do:
```js
for (const directive of directives) {
  const directiveName = directive.name.value;
  const repeatable = repeatableMap[directiveName];
  if (repeatable === undefined || !repeatable) {
    continue;
  }
  // ...
}
```

Reply:
I guess I generally trying to avoid usage of `break` and `continue`, but I can update the code and use it here.

Reply:
@OlegIlyenko Thinking about this can you change `repeatableMap` to `uniqueDirectiveMap` with reverse value and do:
```js 
for (const directive of directives) {
  const directiveName = directive.name.value;
  if (uniqueDirectiveMap[directiveName]) {
    // ...
  }
}
```
Plus `uniqueDirectiveMap` match nicely with `UniqueDirectivesPerLocation` rule name.

Reply:
good idea, done https://github.com/graphql/graphql-js/pull/1541/commits/ae9e7f98f73a069096523575de6869d85707c09f